zetaclient/chains/bitcoin/client/commands.go

```diff
@@ -77,6 +77,41 @@ func (c *Client) GetBlockHeader(ctx context.Context, hash *chainhash.Hash) (*wir
 	return &bh, nil
 }
 
+func (c *Client) GetRawMempool(ctx context.Context) ([]*chainhash.Hash, error) {
```

Comment:
Let's add comment doc for this function

Reply:
https://github.com/zeta-chain/node/pull/3396/commits/3f7bdde1620610dcd6f79b9082860d9e5ca5bf73